vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -0,0 +1,97 @@
+# How to use tabs
+
+This guide show how to effectively utilize tabs to organize and present content within your dashboard.
+
+In the [`Page`][vizro.models.Page] model, you can utilize the `components` argument to incorporate [`Tabs`][vizro.models.Tabs] model.
+
+## Creating a Tabs model
+
+`Tabs` model has `tabs` argument that accepts a list of `Container` models. Each `Container` visually corresponds to an individual tab within the tabbed interface.
+
+
+To create and add tabs to your page, do the following steps:
+
+1. Configure your `components`, see our guide on [Containers](containers.md).
+2. Assign the previously configured `components` to the `tabs` argument of the [`Tabs`][vizro.models.Tabs] model. This creates the connection between your page and the organized tab structure.
```

Comment:
```suggestion
1. Add the [`Tabs`][vizro.models.Tabs] model to the `components` argument of the page you are working
2. Add a list of [`Container`][vizro.models.Container] model to the `tabs` argument of the [`Tabs`][vizro.models.Tabs] model - the length of the list will be the number of tabs
3. Configure the `components` argument for every [`Container`][vizro.models.Container] model, see also our guide on [Containers](containers.md)
```

Would double check the above for details though, but I think like this it is even more prescriptive